### PR TITLE
Prefer non-raw images when determining RG front covers

### PR DESCRIPTION
**Pull request by mwiencek**
https://tickets.metabrainz.org/browse/MBS-13465

### CODE REVIEW

File: docker/docker-compose.test.yml [hunk 1]
```diff
@@ -13,4 +13,4 @@ services:
     environment:
       POSTGRES_HOST_AUTH_METHOD: trust
       POSTGRES_PASSWORD: art
-    image: metabrainz/musicbrainz-test-database:production
+    image: metabrainz/musicbrainz-test-database:v-2024-01-27
```
Comment (yvanzo) [inline docker/docker-compose.test.yml:16]: Is it intended? The second commit seems to be unrelated. At least the commit message doesn’t explain it.

**Review by yvanzo — COMMENTED** [review]
LGTMBDNT. The transactional tests in the first commit seem to be very neat. The actual changes are otherwise similar to https://github.com/metabrainz/musicbrainz-server/pull/3154. Just unsure about how `musicbrainz-test-database` image tags are handled.

File: docker/docker-compose.test.yml (hunk 1 above)
Comment (mwiencek) [reply docker/docker-compose.test.yml:16]: IIRC there were some changes in the SQL inside the image that were needed. I thought that pinning the tag used as done here made sense, since the production tag is updated whenever we rebuild MB docker images.

Comment (yvanzo) [reply docker/docker-compose.test.yml:16]: > IIRC there were some changes in the SQL inside the image that were needed.
OK, was it supposed to be a temporary tag for testing a change that was in another branch?

> the production tag is updated whenever we rebuild MB docker images
Docker Compose will pull the `production` image only if it isn’t locally available, that is for the first run. By default, it won’t pull the image again even if the hub is updated, but it is always possible to ask Docker to check to pull the image again if needed.

Other MeB projects use the tag `production` mainly, or `beta`.

Comment (mwiencek) [reply docker/docker-compose.test.yml:16]: Yes, I remember now that it simply needed the t/sql/initial.sql changes from https://github.com/metabrainz/musicbrainz-server/pull/3154 (which is built into the DB test image). But those changes should be in the production tag now (assuming an old version of the tag isn't cached on Jenkins).

I will try reverting the image tag back to production.

**Review by yvanzo — COMMENTED** [review]
Thank you! :ship:
